e2e/nomostest/testing/ntb.go

```diff
@@ -168,12 +176,13 @@ func (t *FakeNTB) Skipped() bool {
 
 // Log generates the output. It's always at the same stack depth.
 func (t *FakeNTB) Log(args ...interface{}) {
-	fmt.Println(args...)
+	out := fmt.Sprintf("[%s] ", t.name) + fmt.Sprint(args...)
```

Comment:
having the out var doesnt add much here. you can just inline both of these Printlns

Reply:
done